app/views/_volume-claim-templates.html

```diff
@@ -0,0 +1,46 @@
+<div class="pod-template-block">
+  <div
+    ng-repeat="template in templates"
+    class="pod-template">
+    <div class="component-label">Storage claim: {{template.metadata.name}}</div>
+
+    <div row class="pod-template-image icon-row">
+      <div>
+        <span class="fa fa-lock" aria-hidden="true"></span>
+      </div>
+      <div flex class="word-break">
+        <span class="pod-template-key">Access Modes:</span>
+        <span ng-repeat="mode in template.spec.accessModes">
+          {{mode | sentenceCase }}<span ng-if="!$last">, </span>
+        </span>
+      </div>
+    </div>
+
+    <div row class="pod-template-image icon-row">
+      <div>
+        <span class="fa fa-database" aria-hidden="true"></span>
+      </div>
+      <div flex class="word-break">
+        <span class="pod-template-key">Capacity:</span>
+        <span>
+          {{template.spec.resources.requests.storage}}
+        </span>
+      </div>
+    </div>
+
+    <div
+      row class="pod-template-image icon-row"
+      ng-if="template.spec.selector.matchLabels">
```

Comment:
Hmmm. So if I swap that in I get this:

![screen shot 2017-01-19 at 10 37 55 am](https://cloud.githubusercontent.com/assets/280512/22113165/65cea838-de33-11e6-82c6-fd541e219ece.png)

@sg00dwin I think the css in the pod-template targeting `div:first-child` is probably what is doing it, probably too greedy:

```css
.icon-row div:first-child {
    text-align: center;
    width: 30px;
}
```

Pinging you before I update as this is used in a bunch of places.